data/Ewaaugustine.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Ewa Augustine",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student Doctor turned **Software Developer**. Lover of good things and a technical writer",
+  "avatar": "https://github.com/Ewaaugustine.png"
```

Comment:
Hey @Ewaaugustine, as of LinkFree's latest update, we no longer need the `avatar` key. So please remove this avatar field from your file. 

[Updated docs](https://linkfree.eddiehub.io/docs/quickstart)
```suggestion
```